freqtrade/exchange/common.py

```diff
@@ -72,28 +76,34 @@ def calculate_backoff(retrycount, max_retries):
 def retrier_async(f):
     async def wrapper(*args, **kwargs):
         count = kwargs.pop('count', API_RETRY_COUNT)
+        self = args[0]  # Extract the exchange instance.
+        kucoin = self.name == "Kucoin"
         try:
             return await f(*args, **kwargs)
         except TemporaryError as ex:
-            logger.warning('%s() returned exception: "%s"', f.__name__, ex)
+            message = f'{f.__name__}() returned exception: "{ex}"'
+            (log_once_warning if kucoin else logger.warning)(message)
             if count > 0:
-                logger.warning('retrying %s() still for %s times', f.__name__, count)
+                message = f'retrying {f.__name__}() still for {count} times'
+                (log_once_warning if kucoin else logger.warning)(message)
                 count -= 1
-                kwargs.update({'count': count})
+                kwargs['count'] = count
                 if isinstance(ex, DDosProtection):
-                    if "kucoin" in str(ex) and "429000" in str(ex):
+                    if kucoin and "429000" in str(ex):
                         # Temporary fix for 429000 error on kucoin
                         # see https://github.com/freqtrade/freqtrade/issues/5700 for details.
-                        logger.warning(
+                        log_once_warning(
                             f"Kucoin 429 error, avoid triggering DDosProtection backoff delay. "
                             f"{count} tries left before giving up")
                     else:
                         backoff_delay = calculate_backoff(count + 1, API_RETRY_COUNT)
-                        logger.info(f"Applying DDosProtection backoff delay: {backoff_delay}")
+                        message = f"Applying DDosProtection backoff delay: {backoff_delay}"
+                        (log_once_info if kucoin else logger.info)(message)
                         await asyncio.sleep(backoff_delay)
                 return await wrapper(*args, **kwargs)
             else:
-                logger.warning('Giving up retrying: %s()', f.__name__)
+                message = f'Giving up retrying: {f.__name__}()'
+                (log_once_warning if kucoin else logger.warning)(message)
```

Comment:
same applies here - no cached logger for non 429000 messages. 

this is especially true here - which is a non-repeatable message (could for example be an error when placing an order).

Reply:
As above: restored the previous code.